package.json

```
@@ -141,9 +143,9 @@
     "ts-jest": "^28.0.7",
     "ts-loader": "^8.4.0",
```

I think we can bump this up to v9 since that version supports webpack 5
```suggestion
    "ts-loader": "^9.3.1",
```

---

Upgraded